nautobot/ui/jsconfig.json

```diff
@@ -21,7 +21,7 @@
                 "src/components/core/*"
             ],
             "@example_plugin/*": [
-                "/source/examples/example_plugin/example_plugin/ui/*"
+                "../../examples/example_plugin/example_plugin/ui/*"
```

Comment:
TODO Template out

Reply:
This is baking the example plugin by default but that assumes the example plugin is installed and enabled. It shouldn't exist unless it's defined in `PLUGINS`

Reply:
Is this another file that should be gitignored and always regenerated from jsconfig-base.json?